src/query/query-builder.ts

```diff
@@ -0,0 +1,60 @@
+import { ModelFilter, TransactionFilter } from "../filters";
+import { Metadata, QueryParams } from "../types";
+
+export interface BuildPathOptions {
+  filter: ModelFilter | TransactionFilter;
```

Comment:
I haven't pushed everything, you are absolutely right. 
It's because in first approach I tried to build whole URL instead of path only, and then I realized that in endpoints I only specify path not the full url.

Reply:
Fixed